CHANGELOG.md

```diff
@@ -114,6 +114,17 @@ In the future, the component will automatically use the use the `<a>` element if
 
 This change was introduced in [pull request #4646: Deprecate `element` parameter on button component](https://github.com/alphagov/govuk-frontend/pull/4646)
 
+#### Stop using `govuk-body-xs`, `govuk-!-font-size-14` and '14' on the typography scale
+
+We have deprecated '14' on the responsive typography scale as well as classes within GOV.UK Frontend which use 14:
```

Comment:
I'm gonna have a think about this. I want to retain a flat `14` somewhere in there to communicate that it's the 14 point on the scale rather than _necessarily_ 14px (cause it drops down to 12px on small screens). I don't know how actually valuable that is though. Aside: this would be simpler if we had t-shirt sizing aliases set up.

Reply:
I've added something to try and clarify that 14 the scale value ~= 14px the font size. Let me know what you think.

Reply:
Ah I know it doesn't, but we kinda say it does with that [guidance example above](https://github.com/alphagov/govuk-frontend/pull/4649#discussion_r1469308411)

Then in our Sass documentation we call them points?

>Point from the spacing scale (the size as it would appear on tablet and above)

Maybe we embrace that and spell it out as the guidance does?

>We have deprecated 14 point (14px large screens, 12px small screens) on the GOV.UK Frontend responsive typography scale, including font override classes that use 14 point:

Reply:
I've made a light amend to refer to the 14 point as "point 14" just because it feels more natural to me. I'm open to disagreements.